command-parser.js

```diff
@@ -113,6 +113,22 @@ class CommandContext {
 		this.inputUsername = '';
 	}
 
+	checkFormat(room, message) {
+		if (!room) return false;
+		if (!room.stretching && !room.capitals) return false;
+		let formatError = false;
+		let stretchMatch = room.stretching && message.match(/(.+?)\1{7,}/i);
+		let capsMatch = room.capitals && message.match(/[A-Z\s]{18,}/);
```

Comment:
> to avoid filtering out names that are 19 chars in all caps

I don't really think we should have to set it that high.

>where it removes any null characters before doing the stretching check 

Good catch, thanks!